tests/process/wait.rs

```diff
@@ -18,7 +18,7 @@ fn test_waitpid_none() {
         .expect("failed to execute child");
     unsafe { kill(child.id() as _, SIGSTOP) };
 
-    let status = process::waitpid(None, process::WaitOptions::UNTRACED)
+    let (_, status) = process::waitpid(None, process::WaitOptions::UNTRACED)
```

Comment:
Could you add an assert checking that the returned pid matches `child.id()`?

Reply:
Done.